src/flows/publisher_data/update_pocket_score_data_flow.py

```diff
@@ -0,0 +1,196 @@
+            LEFT JOIN content c ON q.content_id = c.id 
+                  AND q.publisher_id = c.publisher_id
+"""
+
+TRUNCATE_CONTENT_QUALITY_SQL = """
+TRUNCATE TABLE content_quality_score_queue
+"""
+
+DATA_FOR_SCORE_CALCULATION_SQL = """
+SELECT 
+    bq.publisher_id, bq.content_id, pcl.save_cnt, pcl.open_cnt, pcl.favorite_cnt, pcl.share_cnt 
+    from `readitla_pub`.content_quality_score_beta_queue bq 
+    left join `readitla_pub`.publisher_content_lifetime pcl on bq.content_id = pcl.content_id 
+          and bq.publisher_id = pcl.publisher_id limit 100
+"""
+
+INSERT_TO_CONTENT_V1_SQL = """
+INSERT IGNORE 
+    INTO `readitla_pub`.content_quality_score (publisher_id, content_id) 
+    values ({publisher_id}, {content_id})
+"""
+
+INSERT_TO_CONTENT_V2_SQL = """
+INSERT IGNORE 
+    INTO `readitla_pub`.content_quality_score_v2 (publisher_id, content_id) 
+    values ({publisher_id}, {content_id})
+"""
+
+UPDATE_CONTENT_V1_SQL = """
+UPDATE `readitla_pub`.content_quality_score 
+    set save_cnt = {save_cnt}, 
+        open_uv = {open_uv}, 
+        open_dv = {open_dv}, 
+        fav_uv = {fav_uv}, 
+        fav_dv = {fav_dv}, 
+        share_uv = {share_uv}, 
+        share_dv = {share_dv}, 
+        open_score = round({open_score},4), 
+        favorite_score = round({fav_score},4), 
+        share_score = round({share_score},4), 
+        total_score = round({total_score},4) 
+        where publisher_id = {publisher_id}
+              and content_id = {content_id}
+"""
+
+UPDATE_CONTENT_V2_SQL = """
+UPDATE `readitla_pub`.content_quality_score_v2 
+    set save_cnt = {save_cnt}, 
+        open_uv = {open_uv}, 
+        open_dv = {open_dv}, 
+        fav_uv = {fav_uv}, 
+        fav_dv = {fav_dv}, 
+        share_uv = {share_uv}, 
+        share_dv = {share_dv}, 
+        open_score = round({open_score},4), 
+        favorite_score = round({fav_score},4), 
+        share_score = round({share_score},4), 
+        total_score = round({total_score},4) 
+        where publisher_id = {publisher_id}
+              and content_id = {content_id}
+"""
+
+DELETE_BETA_QUEUE_SQL = """
+DELETE FROM `readitla_pub`.content_quality_score_beta_queue 
+    where publisher_id = {publisher_id} 
+    and content_id = {content_id}
+"""
+
+
+with Flow(FLOW_NAME) as flow:
+    # max signed int
+    # 2018-04-11 - We hit the max signed int with our item_ids. This broke tables did not have `unsigned` specified.
+    # The hack we are adding here creates a new content_quality_score table (_v2), fixes the column definitions
+    # and inserts new items into that table.
+    max_content_id = 2147483647
+
+    # score variables
+    loss_multiple = 9
+    inc_beta_var = float(1) / (1 + loss_multiple)
+    open_var = 1
+    fav_var = 5
+    share_var = 7
+
+    # prepare variables
+    first_hour_id = 1357020000
+    current_time = int(time.time())
+    today_hour_id = (current_time / 86400) * 86400
+    current_hour_id = today_hour_id - 86400
```

Comment:
```suggestion
```
- I don't see any references to these variables. Can we remove them?
- This code should be in a task to evaluate it at run-time, because otherwise it would be evaluated at build-time.
```python
@task()
def get_time_tuple():
    first_hour_id = 1357020000
    current_time = int(time.time())
    today_hour_id = (current_time / 86400) * 86400
    current_hour_id = today_hour_id - 86400
    return (first_hour_id, current_time, today_hour_id, current_hour_id)
```